runtime/doc/lsp.txt

```diff
@@ -461,6 +461,37 @@ LspSignatureActiveParameter
 ==============================================================================
 EVENTS                                                            *lsp-events*
 
+                                                                   *LspAttach*
+When an LSP client attaches to a buffer. The |autocmd-pattern| is the
+name of the attaching client. Example: >
+
+    vim.api.nvim_create_autocmd("LspAttach", {
+      pattern = "*",
+      callback = function(args)
+        local client = vim.lsp.get_client_by_name(args.match)
```

Comment:
`get_client_by_name` returns a table, right?

(also, need to fix the name when that is decided).

Probably want to filter on `{ buffer = args.buf, name = args.match}` here, too?